.vale/styles/RedHat/SelfReferentialText.yml

```diff
@@ -12,11 +12,3 @@ tokens:
   - this chapter
   - this section
   - this subsection
-  - The following
-  - The previous
-  - This is
```

Comment:
Can we add these to AdjectivesWithNouns? These are other examples (everything from 'this is' to 'those are' but not the last 3, don't include those)

Some "correct" versions would be things like "This is" --> "This function is" 
or 
"Those are" --> "Those instructions are"

Reply:
Done! Updated as suggested!